dev-infrastructure/templates/mgmt-cluster.bicep

```diff
@@ -151,6 +161,16 @@ module dataCollection '../modules/metrics/datacollection.bicep' = {
 //
 // K E Y V A U L T S
 //
+var logsManagedIdentityPrincipalId = filter(mgmtCluster.outputs.userAssignedIdentities, id => id.uamiName == logsMSI)[0].uamiPrincipalID
+
+module logsServiceKeyVaultAccess '../modules/keyvault/keyvault-secret-access.bicep' = {
+  name: guid(cxKeyVaultName, logsMSI, 'certuser')
+  params: {
+    keyVaultName: cxKeyVaultName
```

Comment:
the log cert is supposed to be in the MGMT KV, so `mgmtKeyVaultName` 
see https://docs.google.com/document/d/1YKnMFPFvdIuGpGC1il78O9d3WwTyiVgw7jzCpDTUlII/edit?tab=t.0#heading=h.bupciudrwmna